mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/PrepareGameScreenState.kt

```diff
@@ -1,23 +1,31 @@
 package ch.epfl.sdp.mobile.ui.prepare_game
 
+import ch.epfl.sdp.mobile.application.authentication.AuthenticatedUser
+import ch.epfl.sdp.mobile.application.chess.ChessFacade
+import ch.epfl.sdp.mobile.application.chess.Match
+import ch.epfl.sdp.mobile.ui.social.Person
+import kotlinx.coroutines.CoroutineScope
+
 /**
  * State interface of the [PrepareGameScreen]
  * @property colorChoice chosen color
- * @property gameType chosen game type
+ * @property selectedOpponent the selected opponent whom to create a game with
```

Comment:
I know `@property` was used before but it should not be used, please comment the fields directly instead.

Reply:
Fixed in [3b6d9d6](https://github.com/epfl-SDP/android/pull/200/commits/3b6d9d609f7b69efe8b9d29e5165b1e8cd0b7a60)